addons/stock/models/stock_move.py

```diff
@@ -859,22 +860,16 @@ def onchange_product_id(self):
 
     @api.onchange('lot_ids')
     def _onchange_lot_ids(self):
+        qd = sum(ml.product_uom_id._compute_quantity(ml.qty_done,self.product_uom,False) for ml in self.move_line_ids.filtered(lambda x: x.lot_id.id==False and x.lot_name!=False))
```

Comment:
space after, and operator like == or !=. Try to install flake8 for python. https://pypi.org/project/flake8/
In addition you can find the error code to ignore there https://www.odoo.com/documentation/13.0/reference/guidelines.html#pep8-options